src/storybook/pages/home/home--logged-in.stories.tsx

```diff
@@ -79,7 +79,9 @@ const Template = (args): JSX.Element => (
         <sbb-menu-action>Deutsch</sbb-menu-action>
         <sbb-menu-action>Français</sbb-menu-action>
         <sbb-menu-action>Italiano</sbb-menu-action>
-        <sbb-menu-action icon-name="tick-small">English</sbb-menu-action>
+        <sbb-menu-action icon-name="tick-small" aria-pressed>
```

Comment:
Only works with "true"
```suggestion
        <sbb-menu-action icon-name="tick-small" aria-pressed="true">
```